spec/javascripts/app/models/person_spec.js

```diff
@@ -1,69 +1,69 @@
 
 describe("app.models.Person", function() {
   beforeEach(function() {
-    this.mutual_contact = factory.person({relationship: 'mutual'});
-    this.sharing_contact = factory.person({relationship :'sharing'});
-    this.receiving_contact = factory.person({relationship: 'receiving'});
-    this.blocked_contact = factory.person({relationship: 'blocked', block: {id: 1}});
+    this.mutualContact = factory.person({relationship: "mutual"});
+    this.sharingContact = factory.person({relationship: "sharing'});
+    this.receivingContact = factory.person({relationship: "receiving"});
+    this.blockedContact = factory.person({relationship: "blocked", block: {id: 1}});
```

Comment:
Expected ')' and instead saw '});    this.blockedContact = factory.person({relationship: '.<br>Missing semicolon.<br>Unclosed string.<br>Expected an assignment or function call and instead saw an expression.